src/fontMetrics.js

```diff
@@ -106,7 +106,10 @@ var metrics = {
             return sigma21ScriptScript;
         }
         throw new Error("Unexpected style size: " + style.size);
-    }
+    },
+
+    // \nulldelimiterspace=1.2pt, pg. 344 TeXbook
+    nulldelimiterspace: 1.2 / ptPerEm
```

Comment:
@spicyj I didn't mean to refer to *absolute* font size, I'm fine with the 10 pt per em approach. What I'm worried about is *relative* size: if you go to scriptscriptstyle, then the em in the inner box will be only half as big as it would have been in the outer box. But if you follow LaTeX, then the spaces should not scale by that factor 0.5 but instead should remain fixed. So I'd say that if you have 1.2/ptPerEm outside, then in scriptscriptstyle you should have twice that. Not that it really matters much in this situation here, it's more a matter of principle. Or is there a deliberate decision to also scale such things down, contrary to what LaTeX does?